ExpressWash/ExpressWash/DateFormatter+Clock.swift

```diff
@@ -53,9 +53,7 @@ extension DateFormatter {
 
     static func timeTaken(timeArrived: String?, timeCompleted: String?) -> String {
         guard let timeArrived = timeArrived, let timeCompleted = timeCompleted else { return "In Progress"}
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "hh:mm a"
-        dateFormatter.timeZone = TimeZone.current
+        let dateFormatter = DateFormatter.FromISODate
```

Comment:
You don't actually have to do this. You can just call DateFormatter.FromISODate.whatever if you want.